hapi-fees/src/main/java/com/hedera/services/usage/crypto/CryptoContextUtils.java

```diff
@@ -108,24 +106,10 @@ public static int countSerials(final List<NftAllowance> nftAllowancesList) {
 	}
 
 	static int getNewSerials(
-			final Map<ExtantCryptoContext.AllowanceMapKey, ExtantCryptoContext.AllowanceMapValue> newAllowances,
-			final Map<ExtantCryptoContext.AllowanceMapKey, ExtantCryptoContext.AllowanceMapValue> existingAllowances) {
+			final Map<ExtantCryptoContext.AllowanceMapKey, ExtantCryptoContext.AllowanceMapValue> newAllowances) {
 		int counter = 0;
 		for (var a : newAllowances.entrySet()) {
-			final var isApprovedForAll = a.getValue().approvedForAll().booleanValue();
-			if (existingAllowances.containsKey(a.getKey()) && !isApprovedForAll) {
-				var existingSerials = existingAllowances.get(a.getKey()).serialNums();
-				var newSerials = a.getValue().serialNums();
-				for (var s : newSerials) {
-					if (!existingSerials.contains(s) && s > 0) {
-						counter++;
-					}
-				}
-			} else {
-				if (!isApprovedForAll) {
-					counter += a.getValue().serialNums().size();
-				}
-			}
+			counter += a.getValue().serialNums().stream().distinct().count();
```

Comment:
This feels like it would be more readable if we could write e.g.,
```
int count = 0;
for (final var allowance : newAllowances) {
    count += allowance.uniqueSerialNos();
} 
return count;
```

Reply:
yeah 💯  .. we can just get size of serials. Modified to use `allowance.serialNums().size()`